scripts/known_good.json

```diff
@@ -72,27 +60,19 @@
       "sub_dir" : "Vulkan-ValidationLayers",
       "build_dir" : "Vulkan-ValidationLayers/build",
       "install_dir" : "Vulkan-ValidationLayers/build/install",
-      "commit" : "d86d7a28f94f76804ab272ae86a998f922207799",
+      "commit" : "952b64dbf5931348937635e3459e8811f6533851",
```

Comment:
Is this to depend on VVL PR [659](https://github.com/KhronosGroup/Vulkan-ValidationLayers/pull/659)?
Until that PR is merged, the VVL SHA1 can still change.